docs/src/index.md

```diff
@@ -8,6 +8,76 @@ For more illustrations of its usage in conjunction with other packages, the
+DataFrames.
+
+- **Statistical Modelling / Machine Learning**
+    - [`Statistics`](https://docs.julialang.org/en/v1/stdlib/Statistics/): The Julia standard library comes with a wide range of statistics functionality, but to gain access to these functions you must first load `Statistics`.
+    - [`FreqTables.jl`](https://github.com/nalimilan/FreqTables.jl): Create frequency tables / cross-tabulations. Tightly integrated with DataFrames.
+    - [`HypothesisTests.jl`](https://juliastats.org/HypothesisTests.jl/stable/): A range of hypothesis testing tools.
+    - [`GLM.jl`](https://juliastats.org/GLM.jl/stable/manual/): Tools for estimating linear and generalized linear models. Tightly integrated with DataFrames.
+    - [`statsmodels.jl`](https://juliastats.org/StatsModels.jl/stable/): For converting heterogeneous DataFrames into homogenous matrices for use with linear algebra libraries or machine learning applications that don't directly support DataFrames. Will do things like convert categorical variables into indicators/one-hot-encodings, create interaction terms, etc.
+    - [`MultivariateStats.jl`](https://multivariatestatsjl.readthedocs.io/en/stable/index.html): linear regression, ridge regression, PCA, component analyses tools. Not well integrated with DataFrames, but easily used using `statsmodels.jl`.
+    - [`Clustering.jl`](https://juliastats.org/Clustering.jl/stable/): All forms of clustering algorithms and tools for cluster evaluation.
+    - [`flux.jl`](https://fluxml.ai/Flux.jl/stable/): A pure-Julia package for building your own machine learning models (a little more low-level than `scikit-learn`). Not well integrated with DataFrames, but easily used using `statsmodels.jl`.
+    - [`scikitlearn.jl`](https://scikitlearnjl.readthedocs.io/en/latest/): A Julia wrapper around the full Python `scikit-learn` machine learning library. Not well integrated with DataFrames, but easily used using `statsmodels.jl`.
+- **Plotting**
+    - [`Plots.jl`](http://docs.juliaplots.org/latest/): Powerful, modern plotting library with a syntax akin to that of `matplotlib` (in Python) or `plot` (in R). [`StatsPlots`](http://docs.juliaplots.org/latest/tutorial/#Using-Plot-Recipes-1) provides `Plots.jl` with recipes for many standard statistical plots.
+    - [`gadfly.jl`]: High-level plotting library with a syntax akin to that of `ggplot` (in R).
+    - [`vega-lite.jl`]: High-level plotting library with a syntax akin to that of `altair` (in Python).
+- **Data Wrangling**:
+    - [`DataFramesMeta.jl`](https://github.com/JuliaData/DataFramesMeta.jl): A range of convenience functions for DataFrames that augment `select` and `transform` to provide a user experience similar to that provided by `dplyr` in R.
+    - [`Query.jl`](https://github.com/queryverse/Query.jl): Convenience functions with a similar goal to `DataFramesMeta.jl`, but built to work with any tabular library in Julia (more on other tabular libraries below).
```

Comment:
I think the goals are quite different from `DataFramesMeta`, in particular Query.jl is _not_ tied to tabular data, it just happens to also work well with tabular data. Not sure what a good language here would be, though...

Reply:
How about...:

`Query` provides a single framework for data wrangling that works with a range of libraries, including DataFrames, other tabular data libraries (more on those below), and even non-tabular data. Provides many convenience functions analogous to those in `dplyr` or `linq`.